WordPress/src/main/java/org/wordpress/android/ui/engagement/GetLikesUseCase.kt

```diff
@@ -128,11 +129,17 @@ class GetLikesUseCase @Inject constructor(
                     likes
                 }
 
+                val pageInfo = PagingInfo(
+                        pageLength = paginationParams.pageLength,
+                        page = if (likes.isNotEmpty()) ((likes.size - 1) / paginationParams.pageLength) + 1 else 1,
+                        totalPages = ceil(fingerPrint.expectedNumLikes.toDouble() / paginationParams.pageLength).toInt()
```

Comment:
I'm not sure I understand this one. It seems like this would yield a fatal exception if a zero was ever encountered? Would this be automatically handled by an existing mechanism or would it need some additional handling and logging around it.   Wondering if it should be out of scope here since we're focused on adding analytics?  What do you think?

Reply:
Thanks for clarifying in slack sir!  I've added the changes and included a comment with some of the rationale.